src/borg/remote.py

```diff
@@ -726,6 +727,16 @@ def handle_error(unpacked):
                             return
                 except KeyError:
                     break
+            if cmd == 'barrier' and not self.ignore_responses:
```

Comment:
abusing a command for this seems like a kludge, but maybe it still is the way to go if we want to fix this for 1.1